### PR TITLE
[IMP] web: convert core/network/download tests to hoot

### PR DESCRIPTION
**Pull request by mcm-odoo**
task-id: 3705027

### CODE REVIEW

File: addons/web/static/tests/core/network/download.test.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+            expect.step("file downloaded");
+            document.removeEventListener("click", downloadOnClick);
+            deferred.resolve();
+        }
+    };
+
+    document.addEventListener("click", downloadOnClick);
+    after(() => document.removeEventListener("click", downloadOnClick));
+
+    translatedTerms[translationLoaded] = true; // should always be marked as loaded in tests?
```
Comment (mcm-odoo) [inline addons/web/static/tests/core/network/download.test.js:132]: to check

File: addons/web/tests/test_js.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ class MobileWebSuite(odoo.tests.HttpCase):
     @odoo.tests.no_retry
     def test_unit_mobile(self):
         # Unit tests suite (mobile)
-        self.browser_js('/web/tests/next?headless&tag=-desktop', "", "", login='admin', timeout=1800, error_checker=unit_test_error_checker)
+        self.browser_js('/web/tests/next?headless&tag=-desktop&tag=-headless', "", "", login='admin', timeout=1800, error_checker=unit_test_error_checker)
```
[the remark this answers is not in the document]
Comment (mcm-odoo) [reply addons/web/tests/test_js.py:97]: yes it does, the searchbar adds the tags like `&tag=...&tag=...`.
`&tag=-desktop,-headless` does not seem to work.

**Review by aab-odoo — APPROVED** [review]
robodoo r+

**Comment by mcm-odoo** [comment]
@robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by aab-odoo** [comment]
robodoo retry

**Pull request closed without merging**